pkgs/applications/misc/hcl2json/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "hcl2json";
+  version = "0.3.3";
```

Comment:
Upstream tagged a new version: https://github.com/tmccombs/hcl2json/releases/tag/v0.3.4

Reply:
haha, this is like an hour after I created the PR